src/pages/LandingPage.js

```diff
@@ -67,7 +71,42 @@ const makeDocLinks = _.map(
 )
 
 const LandingPage = () => {
+  const { isSignedIn } = useStore(authStore)
+  const [billingProjects, setBillingProjects] = useState()
+  const signal = useCancellation()
+
+  useEffect(() => {
+    const loadProjects = withErrorHandling(async error => {
+      const errorJSON = await error.json()
+      console.log(`Unable to load billing projects due to: ${errorJSON?.message}`) // eslint-disable-line no-console
+    })(async () => {
+      const projects = await Ajax(signal).Billing.listProjects()
+      setBillingProjects(projects)
+    })
+    if (isSignedIn) {
+      loadProjects()
+    }
+  }, [isSignedIn, setBillingProjects, signal])
+
   return h(HeroWrapper, { bigSubhead: true }, [
+    isTerra() && !_.isUndefined(billingProjects) && _.isEmpty(billingProjects) && div({
+      style: {
+        ...styles.callToActionBanner,
+        width: `calc(${styles.card.width * 3}px + ${styles.card.marginRight} * 2)`,
+        color: '#333f50', backgroundColor: '#e9ebf1', marginBottom: 15
```

Comment:
Why are we using custom colours instead of using one of the appropriate ones in `colors.js`?

Reply:
@yunip20 you can use colors.dark for `color` and colors.light for `backgroundColor`.

Reply:
For the colors in `CreateNewBillingProjectWizard`, I get axe errors about color contrast if I use `colors.accent()` for the step 1 button and radio button texts, and colors.light() for the background of the banners.  I think the warnings also show for the color of link texts. @cahrens I think we wanted to change the colors if the console throws axe warnings. Should I be using other colors in `colors.js` to solve this issue?

Reply:
@yunip20 I will play around with the colors and see what I can do to lighten/darken colors from the palette.

Reply:
Got it, thanks